lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +734,7 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        return tuple(sorted(self._scores, reverse=True, key=lambda key: self._scores[key])[:n_keywords])
```

Comment:
what if several vertices have the same score? how should they be sorted?

Reply:
changed